lib/connextclient/README.md

```diff
@@ -0,0 +1,79 @@
+# Connext
+
+## Installing Connext node for local development
+```bash
+git clone https://github.com/ConnextProject/indra.git
+cd indra
+make
+```
+### Running Connext node
+```bash
+make start
+```
+
+For logs
+```bash
+bash ops/logs.sh node
+```
+
+## Installing Connext REST API client
+```bash
+git clone https://github.com/ConnextProject/rest-api-client
+cd rest-api-client
+npm install
+```
+
+### Running Connext REST API client
+Create environment variables file and point to the Etheruem and Connext node
+
+```bash
+CONNEXT_ETH_PROVIDER_URL="http://0.0.0.0:8545"
+CONNEXT_NODE_URL="http://0.0.0.0:8080"
```

Comment:
Will this conflict with the grpc web proxy which also uses this port by default?